### DISCUSSION TITLE
Possibility to deploy it on a VPS / web hosting?

### DISCUSSION THREAD

**Discussion by Voppis** [balsimpson/alexa-radio#32] [Q&A]
Hello @balsimpson
I like this project very much and I'm using it for more than a year now. Unfortunately, I hit a issue with limits on autocode.com to 100 GB. I want to deploy the code on my own VPS, can you tell me how to do it? I did the git clone but the page is not loading and I see in errors of nginx that it's missing some files.

**Comment by balsimpson** [comment]
I did try and make a version then i lost interest. Here is my code. Not sure if it'll work though.

https://github.com/balsimpson/alexa-radio-skill
